doc/WebSite/Feature-Management.md

```diff
@@ -55,48 +55,48 @@ provider that defines 3 features:
         }
     }
 
-After creating a feature provider, we should register it in our module's
+After creating a feature provider, we must register it in our module's
 [PreInitialize](/Pages/Documents/Module-System#preinitialize) method
 as shown below:
 
     Configuration.Features.Providers.Add<AppFeatureProvider>();
 
 #### Basic Feature Properties
 
-A feature definition requires two properties at least:
+A feature definition requires at least two properties:
 
--   **Name**: A unique name (as string) to identify the feature.
+-   **Name**: A unique name (string) to identify the feature.
 -   **Default value**: A default value. This is used when we need the
-    value of the feature and it's not available for current tenant.
+    value of the feature when it's not available for current tenant.
 
-Here, we defined a boolean feature named "SampleBooleanFeature" which's
-default value is "false" (not enabled). We also defined two value
-features (SampleNumericFeature is defined as child of
-SampleBooleanFeature).
+Here, we defined a boolean feature named "SampleBooleanFeature", with the
+default value of "false" (not enabled). We also defined two value
+features. Note that SampleNumericFeature is defined as a child of
+SampleBooleanFeature.
 
 Tip: Create a const string for a feature name and use it everywhere to
 prevent typing errors.
 
 #### Other Feature Properties
 
-While unique name and default value properties are required, there are
-some optional properties for a detailed control.
+While the unique name and default value properties are required, there are
+some optional properties for more fine-tuned control.
 
--   **Scope**: A value in FeatureScopes enum. It can be **Edition** (if
-    this feature can be set only for edition level), **Tenant** (if this
+-   **Scope**: A value in the FeatureScopes enum. It can be and **Edition** (if
```

Comment:
`and` (even if you meant `an` here) reads wrong. **Edition**, **Tenant** and **All** are enum values, so an *article* is not appropriate.